crafters/nlp/PDFExtractorSegmenter/tests/test_pdfextractorcrafter.py

```diff
@@ -12,97 +12,69 @@
 path_img = os.path.join(cur_dir, 'cats_are_awesome_img.pdf')
 
 
-def test_io_uri_images_and_text():
-    crafter = PDFExtractorSegmenter()
-    chunks = crafter.craft(uri=path_img_text, buffer=None)
+def test_images_and_text():
+    for i in range(2): #Test for URI and buffer
```

Comment:
this that u want to achieve is achieved with `@pytest.mark.parametrize` (see TransformTorchEncoder for instance)

Reply:
but just keep 2 separate tests since the name of the `tests` are good hint when the test fails. A test should do as much as possible one and only one thing